lib/tapioca/gem/pipeline.rb

```diff
@@ -161,8 +161,10 @@ def on_symbol(event)
         symbol = event.symbol.delete_prefix("::")
         return if symbol_in_payload?(symbol) && !@bootstrap_symbols.include?(symbol)
 
-        constant = constantize(symbol)
-        push_constant(symbol, constant) if constant
```

Comment:
To be honest, I don't know why we use `const_get` instead of `const_defined?`. @paracycle any ideas on this?

Reply:
We don't need `const_defined?` inside the `constantize` method because the expectation is that the operation will succeed and it not succeeding is an exception.

If we did a `const_defined?` followed by a `const_get` every time, that would slow down the common path (since it might end up making the const lookup twice).

We just need to handle the exception more gracefuly.